docs/source-pytorch/advanced/warnings.rst

```diff
@@ -0,0 +1,52 @@
+########
+Warnings
+########
+
+Lightning warns users of possible misconfiguration, performance implications or potential mistakes through the :class:`~lightning.pytorch.utilities.warnings.PossibleUserWarning` category.
```

Comment:
just confused about the class name `PossibleUserWarning`, would be be `UserSuggestions`?

Reply:
This warning category is meant to express that the warning may or may not be a false positive, and the user may need to ignore it for valid reasons. The name can't really be changed at the moment even if there was a better name, because it would be a breaking change for users who already have ignore-rules.

Reply:
> The name can't really be changed at the moment even if there was a better name

:cry: 

`PotentialUserWarning`?